LICENSE.md

```diff
@@ -1 +1,295 @@
-For licensing details please consult https://github.com/whosonfirst/whosonfirst-data/blob/master/LICENSE.md
+# License
+
+For a full list of sources used in Who's On First, please visit [this page](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/README.md).
+
+Crediting Who's On First is recommended and linking back to the License is required. For example:
+
+> Data from Who's On First. [License](https://whosonfirst.org/docs/licenses/).
+
+The Who's On First dataset is both original work and a modification of existing open data (sources listed below). Some of those open data projects do require attribution. We detail all sources and their specific license, usage, and vintage metadata in the full [sources list](https://whosonfirst.org/docs/licenses/).
+
```

Comment:
Please also include the `_This file was generated by the `bin/docs.py` script on 2020-02-20._` line here-ish.

Reply:
fixed in 5827205